test/wh_test_clientserver.c

```diff
@@ -458,6 +458,94 @@ static int _testDma(whServerContext* server, whClientContext* client)
     return rc;
 }
 
+int _testClientCounter(whClientContext* client)
+{
+    const whNvmId  counterId              = 1;
+    const uint32_t MAX_COUNTER_VAL        = 0xFFFFFFFF;
```

Comment:
Grumble.  This should be in the counter header file somewhere.  Probably should be overridable at creation.  I'll add this to the counter backlog item.  No change.